src/kbmod/work_unit.py

```diff
@@ -75,9 +76,13 @@ def __init__(
         geocentric_distances=None,
         reprojected=False,
         per_image_indices=None,
+        lazy=False,
+        file_paths=None,
```

Comment:
How does using file_paths overlap with what `ImageCollection` is doing when getting things? Is there anything there we could reuse?

Reply:
I think the `file_paths` for `WorkUnit` here specifically refer to files generated by a WorkUnit as part of sharding, and my assumption is that we would want to keep it that way as there is a one-to-many relationship between `ImageCollection` URIs and `WorkUnit`s, but there might be some wisdom in making `WorkUnit` loading even lazier (esp. when we're not doing reprojection) 😅